packages/app-staking/src/md/basic.md

```diff
@@ -0,0 +1,32 @@
+# Staking
+
+Welcome to the staking module. Here you can see an overview of all the validators on the network and participate in the network as either as validtor (running a node that helps secure the network), or a nominator (adding funds to help secure the network).
+
+Stakers are nominators earn rewards: at the end of an era the rewards are split between the validators and nominators for that validator, in proportion to the amount bonded by each individual. Simmilarly, at the end of a session set of new validators are selected.
+
+# Bonding
+
+Bonding funds is the first step that either a validator or nominator performs. It instructs the network to lock up a portion of funds that is used for security. As a validator or nominator, these funds are placed at risk, i.e. you can be slashed and lose a portion if the validator node misbehaves. Similarly, as a nominator, choosing a misbehaving validator will also have a slashing effect. At the same time, should the validators perform well, rewards are payed to the validators and nominators for authoring.
```

Comment:
I think we don't need to have the notion of "instructing the network" to abstract the fact that those are transactions.

```suggestion
Bonding funds is the first step that either a validator or nominator performs. It locks up a portion of funds that is used to secure the network. These funds are placed at risk, i.e. you can be slashed and lose a portion if the validator node misbehaves. Validators and nominators share both rewards and the slashing effects. Choosing a well behaving validator to nominate is crucial.
```